ci-operator/step-registry/openshift/upgrade/qe/sanity/arch-migration/openshift-upgrade-qe-sanity-arch-migration-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: openshift-upgrade-qe-sanity-arch-migration
+  steps:
+  - ref: cucushift-upgrade-prehealthcheck
+  - ref: cucushift-ota-preupgrade
+  - ref: cucushift-upgrade-arch-migration
+  - ref: cucushift-ota-postupgrade
```

Comment:
Hmm, what test are there in this step? Should we run it after `ref: ipi-install-heterogeneous`?

Reply:
> what test are there in this step

Searching `ENABLE_OTA_TEST` keyword, `OCP-60396`, `OCP-60397` and `OCP-53921` are enabled on these jobs.

> Should we run it after ref: ipi-install-heterogeneous

Actually I copied the steps from openshift-upgrade-qe-test-arch-migration chain, I just removed some useless e2e steps, I did not change the order.  Actually I do not have much concern on that, because for OTA testing, mainly happened after `oc adm upgrade` command. And at least, before we run the OTA testing following the order.

Reply:
Got it. Thank you!